recipes/ngseqbasic/meta.yaml

```diff
@@ -21,8 +21,10 @@ requirements:
     - ucsc-bedclip ==332
     - ucsc-bedgraphpack ==332
     - ucsc-bedtobigbed ==332
+    - ucsc-bigbedtobed ==332
```

Comment:
I would remove those strict pinnings in general. To guarantee a certain environment you should use environment.yaml files instead.